lxd/devices.go

```diff
@@ -169,7 +170,11 @@ func findNvidiaMinor(pci string) (string, error) {
 	return "", err
 }
 
-func deviceLoadGpu() ([]gpuDevice, []nvidiaGpuDevices, error) {
+func wantsAllGPUs(m map[string]string) bool {
```

Comment:
Nit but `deviceWantsAllGPUs` instead? trying to keep things moderately namespaced.